src/views/AddItem.jsx

```diff
@@ -20,8 +21,23 @@ export function AddItem({ listToken }) {
 		setItem({ ...item, [e.target.name]: updateVal });
 	};
 
+	const isInvalid = (name) => {
+		if (isEmpty(name)) {
+			setStatus('Can not add an empty item');
```

Comment:
nit- 'Cannot' or 'Uh oh! It looks like you didn't enter a grocery item'

Reply:
Changed !